src/frontend/prettier.config.js

```diff
@@ -0,0 +1,10 @@
+/** @type {import("prettier").Config} */
+const config = {
+  trailingComma: "es5",
+  tabWidth: 2,
+  semi: false,
+  singleQuote: true,
```

Comment:
I would prefer to require double quotes. What do you think?

Reply:
Agreed about double quotes. Will also set `semi` to `true`.